fixups/RegLegacyFixups/RegistryFixups.cpp

```diff
@@ -45,10 +52,10 @@ std::string ReplaceRegistrySyntax(std::string regPath)
     {
         if (regPath.length() > 18)
         {
-            size_t offsetAfterSid = regPath.find('\\', 19);
+            size_t offsetAfterSid = regPath.find('\\', 18);
```

Comment:
Wouldn't this find offsetBeforeSid instead of offsetAfterSid

Reply:
Do we get Sid in HKLM path?

Reply:
Agreed, it looks \Registry\Machine entries do not have SID which is contradictory from the variable name 'offsetAfterSid'.